hles/extraction/src/test/scala/org/broadinstitute/monster/dap/HLESExtractionPipelineBuilderIntegrationSpec.scala

```diff
@@ -69,4 +69,37 @@ class HLESExtractionPipelineBuilderIntegrationSpec extends PipelineBuilderSpec[A
   it should "successfully download records from RedCap" in {
     readMsgs(hlesOutputDir, "records/*.json") shouldNot be(empty)
   }
+
+  val fakeIds = 0 to 1
```

Comment:
It'd be good to tie this with the expectedStudyIds val we are declaring on line 28, so we can avoid future test breakage if we change the set of IDs. What if we generate the expectedStudyIds val from this list?